zshrc

```diff
@@ -78,6 +78,30 @@ export PATH="/usr/local/bin:$PATH"
 export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
 
+autoload -U add-zsh-hook
+load-nvmrc() {
+  local node_version;
+  local nvmrc_path;
+  node_version="$(nvm version)"
+  nvmrc_path="$(nvm_find_nvmrc)"
+
+  if [ -n "$nvmrc_path" ]; then
+    local nvmrc_node_version;
+    nvmrc_node_version=$(nvm version "$(cat "${nvmrc_path}")")
+
+    if [ "$nvmrc_node_version" = "N/A" ]; then
+      nvm install
```

Comment:
I think there is a discussion to be had here, we may rather just inform the user, as I could see this behavior being rather annoying.

Reply:
I can see this causing some confusion from developers that are not expecting autoload node versioning. 

Just spitballing, what do you think of adding an env var to disable? If it becomes annoying, they have the option of turning it off (like like `npm sponsor`).

Reply:
They can customize their option fully in `~/.zshrc.user` for example if they do not want this behavior they can disable the hook entirely: `add-zsh-hook -d chpwd load-nvmrc`